GNUmakefile

```diff
@@ -12,7 +12,7 @@ WARNINGS = -Wall -Wextra -Wpedantic -Wformat=2 -Wstrict-aliasing=3 -Wstrict-over
 	-Wfloat-equal -Wcast-align -Wpointer-arith -Wchar-subscripts -Warray-bounds=2 -Wno-unknown-warning-option
 
 override CFLAGS ?= -g -O2 $(WARNINGS)
-override CFLAGS += -std=c99
+override CFLAGS += -std=c99 -D_POSIX_C_SOURCE=200809L
```

Comment:
this should go to CPPFLAGS instead, also does this conflict with -D_DEFAULT_SOURCE?

Reply:
Oh, I didn't see that you already have `_DEFAULT_SOURCE` enabled. The [documentation for `_DEFAULT_SOURCE`](https://www.gnu.org/software/libc/manual/html_node/Feature-Test-Macros.html#index-_005fDEFAULT_005fSOURCE) states the following:

> If you define this macro, most features are included apart from X/Open, LFS and GNU extensions: the effect is to enable features from the 2008 edition of POSIX, as well as certain BSD and SVID features without a separate feature test macro to control them. 

Since this macro already enables the POSIX.1-2008 features, we don't need to set `_POSIX_C_SOURCE`. I don't actually use glibc, so please check on your system, but I did some quick tests on Debian which confirm this.

Removed the Makefile modifications and adjusted the commit message.